main.go

```diff
@@ -153,22 +153,49 @@ func Initialize(e *echo.Echo, api APIInterface, apiConfig string) *echo.Echo {
 						}
 						switch method {
 						case "GET":
```

Comment:
Looking at the implementation of EnableCORS I think it makes sense to first looking at the options available for the path and then sending as an array (instead of doing it in every case in the switch)

Reply:
Upon thinking on this further it's applying it per path which should be fine